src/Datadog.Monitoring.Distribution/Datadog.Monitoring.Distribution.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <Version>1.28.5-prerelease</Version>
+    <Version>$(Version)-beta01</Version> <!-- Unconditionally add a beta suffix to the package, but keep the rest of the name so we can associate it with the rest of the release -->
```

Comment:
Could this be done with `<VersionSuffix>` instead? Seems a little cleaner that using `<Version>` twice.

See this blog some "some guy" wrote: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/

Reply:
>setting Version explicitly will override any `VersionPrefix` or `VersionSuffix` settings.

Ehh. Looks like we would need to use  `<VersionPrefix>` and `<VersionSuffix>` for this to work.

Reply:
Yeah, this approach works and it creates the smallest amount of changes, so I'd like to keep this approach